internal/core/items/text.rs

```diff
@@ -232,7 +233,7 @@ impl Text {
             },
             pixel_size: {
                 let font_size = self.font_size();
-                if font_size == 0.0 {
+                if font_size.approx_eq(&0.0) {
```

Comment:
```suggestion
                if font_size <= 0. {
```

negative font-size don't make sense